Public/Src/Sandbox/MacOs/Interop/Sandbox/ESSandbox.cpp

```diff
@@ -158,10 +150,26 @@ extern "C"
         
         // Subsribe and activate the ES client
         
-        es_return_t status = es_subscribe(client, sandbox->GetSubscibedESEvents(), sandbox->GetSubscribedESEventsCount());
+        es_return_t status = es_subscribe(client, sandbox->GetEventsForType(EventType::ProcessEvent), sandbox->GetEventCountForType(EventType::ProcessEvent));
```

Comment:
>ndbox->GetEventCountForType(EventType:: [](start = 104, length = 39)

Just curious: Would it make sense to store this in a variable and use in the error messages to help diagnose some of the errors below?